hal/targets/hal/TARGET_STM/TARGET_STM32L0/rtc_api.c

```diff
@@ -39,11 +40,29 @@ static int rtc_inited = 0;
 
 static RTC_HandleTypeDef RtcHandle;
 
+#if DEVICE_RTC_LSI
```

Comment:
Lets reserve DEVICE_ for magic in the device_has list. other macros can be STM_RTC_LSE etc.